docker-compose.yaml

```diff
@@ -5,6 +5,21 @@ networks:
     driver: bridge
 
 services:
+  reverse-proxy:
+    image: "traefik:v3.3"
+    container_name: "traefik"
+    ports:
+      - 80:80
+      - 443:443
+      - 8080:8080
+    volumes:
+      - /var/run/docker.sock:/var/run/docker.sock:ro
+      - ./conf/traefik.yaml:/etc/traefik/traefik.yaml
+      - ${HOME}/.certs/:/certs/
```

Comment:
I’d like to hear your opinion on whether there’s a better path for storing this TLS certificate.

Would ~/.cloud-barista/certs be a good choice to avoid conflicts with other applications?